src/collections/_documentation/workflow/notifications/workflow.md

```diff
@@ -3,12 +3,99 @@ title: 'Workflow'
 sidebar_order: 2
 ---
 
-Most activity within Sentry will generate a workflow notification. Core workflow notifications include:
+Most activity within Sentry will generate a workflow notification. You can also configure various personal notifications from your [personal notifications settings](https://sentry.io/settings/account/notifications) and any specific project's settings in the Project Settings page.
 
--   Assignment
--   Comments
--   Regressions – when Sentry changes state of an issue from **resolved** to **unresolved**
+By default, these notifications are sent to anyone who is a member of the project the issue occurs in. Still, each member may choose their participation state, as well as opt-in or opt-out of notifications for specific issues. See [Unsubscribe](#unsubscribe) below for more details.
 
-By default, these notifications are sent to anyone who is a member of the project the issue occurs in, but each member may choose their participation state, as well as opt in or opt out of notifications for specific issues (in the "Notifications" section at the bottom right of the issue detail view). 
+## Issue States
 
-Read about [notification management]({%- link _documentation/workflow/notifications/notification-management.md -%}) for more information.
+Sentry provides a few states for each issue, which significantly impacts how notifications work.
+
+### Unresolved
+
+- The default state when an issue is added to the system.
+
+### Resolved
+
+- An issue is marked as resolved when an individual resolves it by manually changing the state on the issue page UI, submitting a fix, or when the project’s auto-resolve feature is configured. Team members are notified via email. Auto-resolve is located in **Project Settings > General Settings**.
+
```

Comment:
could we also add (on the main page as well where this is duplicated) what the values in the slider are? it's either disabled or between 1 hour and 30 days. I can get the full breakdown of options if you want, but it might be overkill.

Reply:
Hmmm... maybe? Do you want to make a suggested edit? That way I can see what you mean?